config/software/openssl-customization.rb

```diff
@@ -0,0 +1,68 @@
+#
+# Copyright:: Copyright (c) 2014 Chef Software, Inc.
+# License:: Apache License, Version 2.0
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+# This software makes sure that SSL_CERT_FILE environment variable is pointed
+# to the bundled CA certificates that ship with omnibus. With this, Chef
+# tools can be used with https URLs out of the box.
+name "openssl-customization"
+
+source path: "#{project.files_path}/#{name}"
```

Comment:
@tylercloke I think this should work: delete this line https://github.com/chef/omnibus-software/pull/403/files#diff-52d77087eeaef44fe419c4a5d0f3b2b3R37 to remove the chef software defn's dep on openssl-customization, and then move openssl-customization back to omnibus-chef, and add a dep on openssl-customization to the chef and chefdk ( @schisamo @jdmundrawala -- anything else, angry chef?) _project_ defs.